### PR TITLE
Add missing dependency on python3-bitstring to python-requirements

### LINKED ISSUE #930: Missing dependency on python3-bitstring
The riscv-dv code uses it and I don't think we have it in python-requirements.

### PR DESCRIPTION
**Pull request by rswarbrick**
Closes #930.

### CODE REVIEW

**Comment by udinator** [comment]
We already include the `bitstring` package in the requirements.txt file in the `riscv-dv` repository, should it be duplicated here also?

File: python-requirements.txt [hunk 1]
```diff
@@ -13,3 +13,6 @@ mako
 
 # Needed by dvsim.py (not actually used in Ibex)
 premailer
+
+# Needed by riscv-dv
+bitstring
```
Comment (imphil) [inline python-requirements.txt:18]: Following @udinator's sugggestion, we can do that:

```suggestion
-r vendor/google_riscv-dv/requirements.txt
```

Comment (rswarbrick) [reply python-requirements.txt:18]: Oh, neat! I didn't know that pip did that. I'll push up an amended version.

**Comment by rswarbrick** [comment]
Yeah. For now, the requirements won't be vendored in. Maybe we should change the OT top-level to include a `util/dvsim/python-requirements.txt`, which would let us do that?

**Comment by imphil** [comment]
Something along those lines would be nice one day, but there's no immediate need to do that right now.

**Pull request merged**